templates/home.html

```diff
@@ -4,25 +4,40 @@
 
 {% block content %}
 
-<div class="container justify-content-center col-lg-8">
-    <p class="h2 text-center">Welcome to Quixr!</p>
+<div class="container pl-4 pr-4 mt-2 mb-3 col-lg-8">
+    <div class="row d-flex justify-content-center">
+        <p class="h2 text-info">Welcome to Quixr!</p>
```

Comment:
A11y note: would be more semantically "good" to make it an actual `<h2>`.

Reply:
:+1: On it.